contrib/tvmop/compile.py

```diff
@@ -152,6 +152,12 @@ def get_cuda_arch(arch):
     # we create libtvmop.o first, which gives us chance to link tvm_runtime together with the libtvmop
     # to allow mxnet find external helper functions in libtvm_runtime
     func_binary.save(arguments.target_path + "/libtvmop.o")
+    try:
+        imported_modules
+    except NameError:
+        imported_modules = []
+    if len(imported_modules):
```

Comment:
why checking `imported_modules` while actually accessing `func_binary.imported_modules` ?

Reply:
Sorry, I fixed it.